### ISSUE TITLE
Bitcentive build fails on Node 7.2.1

### ISSUE THREAD

**Issue by marshallswain** [stealjs/steal-tools#575]
The Bitcentive build works in Node 6.5.0, but not with the latest 7.2.1:

- Here's the [build file](https://github.com/donejs/bitcentive/blob/staging/public/build.js)
- And the [package.json](https://github.com/donejs/bitcentive/blob/staging/public/package.json)

Everything seems to be in place in the node_modules folder.
```
Bitovis-MacBook-Pro:bitcentive marshall$ node public/build
OPENING: /Users/marshall/Sites/bitcentive/public/package.json!npm
Potentially unhandled rejection [8] Error: Error loading "can-set@1.0.3#src/set" at file:/Users/marshall/Sites/bitcentive/public/can-set@1.0.3#src/set.js
Error loading "can-set@1.0.3#src/set" from "bitcentive@0.0.1#index.stache!done-autorender@0.9.0-pre.1#autorender" at file:/Users/marshall/Sites/bitcentive/public/index.stache
Error loading "can-set@1.0.3#src/set" at file:/Users/marshall/Sites/bitcentive/public/can-set@1.0.3#src/set.js
Error loading "can-set@1.0.3#src/set" from "bitcentive@0.0.1#pages/dashboard.component-template" at file:/Users/marshall/Sites/bitcentive/public/pages/dashboard.component.template
ENOENT: no such file or directory, open '/Users/marshall/Sites/bitcentive/public/can-set@1.0.3#src/set/index.js'
Potentially unhandled rejection [11] Error: Error loading "can-set@1.0.3#src/set" at file:/Users/marshall/Sites/bitcentive/public/can-set@1.0.3#src/set.js
Error loading "can-set@1.0.3#src/set" from "bitcentive@0.0.1#index.stache!done-autorender@0.9.0-pre.1#autorender" at file:/Users/marshall/Sites/bitcentive/public/index.stache
Error loading "can-set@1.0.3#src/set" at file:/Users/marshall/Sites/bitcentive/public/can-set@1.0.3#src/set.js
Error loading "can-set@1.0.3#src/set" from "bitcentive@0.0.1#pages/dashboard.component-template" at file:/Users/marshall/Sites/bitcentive/public/pages/dashboard.component.template
ENOENT: no such file or directory, open '/Users/marshall/Sites/bitcentive/public/can-set@1.0.3#src/set/index.js'
Potentially unhandled rejection [19] AssertionError: loading or loaded
    at Console.assert (console.js:95:23)
    at /Users/marshall/Sites/bitcentive/public/node_modules/steal-es6-module-loader/dist/es6-module-loader.src.js:1495:17
    at tryCatchReject (/Users/marshall/Sites/bitcentive/public/node_modules/steal-es6-module-loader/dist/es6-module-loader.src.js:1183:30)
    at runContinuation1 (/Users/marshall/Sites/bitcentive/public/node_modules/steal-es6-module-loader/dist/es6-module-loader.src.js:1142:4)
    at Fulfilled.when (/Users/marshall/Sites/bitcentive/public/node_modules/steal-es6-module-loader/dist/es6-module-loader.src.js:930:4)
    at Pending.run (/Users/marshall/Sites/bitcentive/public/node_modules/steal-es6-module-loader/dist/es6-module-loader.src.js:821:13)
    at Scheduler._drain (/Users/marshall/Sites/bitcentive/public/node_modules/steal-es6-module-loader/dist/es6-module-loader.src.js:97:19)
    at Scheduler.drain (/Users/marshall/Sites/bitcentive/public/node_modules/steal-es6-module-loader/dist/es6-module-loader.src.js:62:9)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
Potentially unhandled rejection [20] AssertionError: loading or loaded
    at Console.assert (console.js:95:23)
    at /Users/marshall/Sites/bitcentive/public/node_modules/steal-es6-module-loader/dist/es6-module-loader.src.js:1495:17
    at tryCatchReject (/Users/marshall/Sites/bitcentive/public/node_modules/steal-es6-module-loader/dist/es6-module-loader.src.js:1183:30)
    at runContinuation1 (/Users/marshall/Sites/bitcentive/public/node_modules/steal-es6-module-loader/dist/es6-module-loader.src.js:1142:4)
    at Fulfilled.when (/Users/marshall/Sites/bitcentive/public/node_modules/steal-es6-module-loader/dist/es6-module-loader.src.js:930:4)
    at Pending.run (/Users/marshall/Sites/bitcentive/public/node_modules/steal-es6-module-loader/dist/es6-module-loader.src.js:821:13)
    at Scheduler._drain (/Users/marshall/Sites/bitcentive/public/node_modules/steal-es6-module-loader/dist/es6-module-loader.src.js:97:19)
    at Scheduler.drain (/Users/marshall/Sites/bitcentive/public/node_modules/steal-es6-module-loader/dist/es6-module-loader.src.js:62:9)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
WARN: can-view-parser: Found attribute with name:  viewBox . Converting to:  view-box
WARN: can-view-parser: Found attribute with name:  viewBox . Converting to:  view-box
WARN: can-view-parser: Found attribute with name:  viewBox . Converting to:  view-box
WARN: can-view-parser: Found attribute with name:  viewBox . Converting to:  view-box
WARN: can-view-parser: Found attribute with name:  viewBox . Converting to:  view-box
WARN: can-view-parser: Found attribute with name:  viewBox . Converting to:  view-box
WARN: can-view-parser: Found attribute with name:  viewBox . Converting to:  view-box
WARN: can-view-parser: Found attribute with name:  viewBox . Converting to:  view-box
WARN: can-view-parser: Found attribute with name:  viewBox . Converting to:  view-box
WARN: can-view-parser: Found attribute with name:  viewBox . Converting to:  view-box
WARN: can-view-parser: Found attribute with name:  viewBox . Converting to:  view-box
WARN: can-view-parser: Found attribute with name:  viewBox . Converting to:  view-box
WARN: can-view-parser: Found attribute with name:  viewBox . Converting to:  view-box
WARN: can-view-parser: Found attribute with name:  viewBox . Converting to:  view-box
WARN: can-view-parser: Found attribute with name:  viewBox . Converting to:  view-box
```

**Comment by marshallswain** [comment]
This was fixed by upgrading to the latest 1.x.x releases of steal and steal-tools.  I was running a prerelease which was working fine until today.

**Comment by marshallswain** [comment]
There's still something weird going on here. It looks like there is maybe an issue in Steal. When it loads can-set, the URL in the browser is `http://localhost:3030/can-set@1.0.3%23src/set.js`.  I reinstalled node_modules with node `6.5.0` and the issue went away.

**Comment by marshallswain** [comment]
My Travis build finally passed when I locked node at `6.5.0`.  I think the problem even exists when using `6.9.2`.

**Comment by matthewp** [comment]
Can you try again? There was a race condition when loading plugin configuration that was fixed in Steal 1.0.4, this looks like it could be related to that.

**Comment by matthewp** [comment]
Going to close for now, reopen if you test it and still have an issue.
